src/editors/utils/convertMarkdownToXML.js

```diff
@@ -0,0 +1,541 @@
+const convertMarkdownToXml = (markdown) => {
```

Comment:
I guess this isn't really a new file and it may be too late anyways so I won't block on this, but when adding new files in any of our frontend repos, please use TypeScript `.ts[x]` instead of `.js[x]` ! Especially something with this much complicated logic.